client/components/charts/locations/_clients_chart.html

```diff
@@ -3,7 +3,7 @@
     <md-card-header-text>
       <!-- <span ng-if="resource == 'device'" class="md-subhead" translate>Device Usage</span> -->
       <!-- <span ng-if="resource != 'device'" class="md-subhead" translate>WiFi Usage</span> -->
-      <span class="md-subhead" translate>Users</span>
+      <span class="md-subhead" translate>Clients</span>
```

Comment:
For the time being, we're sticking with users and will discuss internally whether this is ok.